getting-started.md

```diff
@@ -59,3 +59,23 @@ $ exogress spawn --access-key-id=<ACCESS_KEY_ID> --secret-access-key=<SECRET_ACC
 ```
 
 That's it. Now the server on port `3000` is available under the selected domain.
+
+## Advanced
+
+### Project-level config
+
+Project-level config simplifies configuration in cases when you have multiple client-level configs and want to set some common behaviour across all of them. Does not support upstreams.
```

Comment:
The main difference is that it works even without any running clients. Lack of upstream is not the only difference. E.g. proxy and static-dir handlers are also disallowed. Project config clearly needs an additional section with more information.